base/debian/control

```diff
@@ -35,7 +35,7 @@ Depends: adduser,
          ebtables-legacy | ebtables,
          f2fs-tools,
          grub-efi-arm64-bin [arm64],
-         grub-efi-ia32-bin | grub-efi-amd64-bin [!arm64],
+         grub-efi-ia32-bin [!arm64] | grub-efi-amd64-bin [!arm64],
```

Comment:
A comment on the other PR mentioned that depending on the amd64 version was wrong: https://github.com/google/android-cuttlefish/pull/615#issuecomment-2233833959

We should just revert it to depending on the `ia32` package only.

Reply:
Please revert it, because this blocks the ARM64 image/packages build. Thanks!